File: 0782-jewels-and-stones/0782-jewels-and-stones.js
/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function(jewels, stones) {
    let jewCnt = 0;
    let stoAr = stones.split('');
    for(let i=0;i<stoAr.length;i++) {
        if(jewels.includes(stoAr[i])) jewCnt++;
    }
    return jewCnt;
};

File: 0782-jewels-and-stones/0782-jewels-and-stones.cs
public class Solution {
    public int NumJewelsInStones(string jewels, string stones) {
        int jewCnt = 0;
        char[] stAr = stones.ToCharArray();        
        foreach(var ch in stAr) {
            if(jewels.Contains(ch)) jewCnt++;
        }
        return jewCnt;
    }
}